scenarioo-client/app/step/step.controller.ts

```diff
@@ -298,18 +299,21 @@ function StepController($scope, $routeParams, $location, $route, StepResource, S
     });
 
     function loadRelatedIssues() {
-        RelatedIssueResource.query({
+        RelatedIssueResource.get({
             branchName: SelectedBranchAndBuildService.selected().branch,
             buildName: SelectedBranchAndBuildService.selected().build,
+            },
             useCaseName,
             scenarioName,
-            pageName: $scope.pageName,
-            pageOccurence: $scope.pageOccurrence,
-            stepInPageOccurrence: $scope.stepInPageOccurrence,
-        }, (result) => {
-            $scope.relatedIssues = result;
+            $scope.pageName,
+            $scope.pageOccurrence,
+            $scope.stepInPageOccurrence,
+        ).subscribe((relatedIssueSummary: RelatedIssueSummary[]) => {
+            $scope.relatedIssues = relatedIssueSummary;
             $scope.hasAnyRelatedIssues = () => $scope.relatedIssues.length > 0;
```

Comment:
Same issue as previously. relatedIssues can be null, which leads to JavaScript errors.